cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,7 @@ class AddressStepPage {
     private addressButton: string;
 
     constructor() {
-        this.addressButton = ".cart_navigation > .button > span";
+        this.addressButton = ".cart_navigation .button";
```

Comment:
`button[name='processAddress']` is better

Reply:
all done @holgiosalos